build.gradle

```diff
@@ -25,12 +25,17 @@ import java.util.function.Predicate
 import java.util.concurrent.TimeUnit
 import java.util.stream.Collectors
 import org.opensearch.gradle.testclusters.OpenSearchCluster
+import org.opensearch.gradle.test.RestIntegTestTask
 
 buildscript {
 
     ext {
         isSnapshot = "true" == System.getProperty("build.snapshot", "false")
         opensearch_version = System.getProperty("opensearch.version", "1.2.0-SNAPSHOT")
+        // for bwc tests
+        opensearch_previous_version = "1.1.0-SNAPSHOT"
+        plugin_previous_version = "1.1.0.0-SNAPSHOT"
```

Comment:
We always test upgrade from previous version is it? (when 1.3 comes, the tests will upgrade from 1.2 to 1.3, correct?)

If so, do we anticipate the need to also cover 1.1 to 1.3?

Reply:
I've hardcoded this here because we need the security & replication plugin artefact locally present for the older versions. These are not available in the repo yet: https://aws.oss.sonatype.org/content/repositories/snapshots/org/opensearch/
After these are availabe, we can run for any combination possible. For now, I'll just add the support to pass this as param.